frontend/src/views/SignUpView.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div>
+    <h1>S'Inscrire</h1>
+    <form
+      @submit.prevent="signup"
+      style="border: solid 1px black; width: 25vw; min-width: 300px"
```

Comment:
You shouldn't use style inside html template.. it should go inside <style></style> tag at the end of the file

Reply:
Alright I'll move it to the bottom.